packages/react-next/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -7,7 +7,7 @@ import { KeyboardSpinDirection } from './SpinButton';
 import { IButtonStyles } from '../../Button';
 import { IKeytipProps } from '../../Keytip';
 import { IRefObject } from '../../Utilities';
-import { IButtonProps } from 'office-ui-fabric-react/lib/components/Button/Button.types';
+import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
```

Comment:
After #13575 you should change this to:
```suggestion
import { IButtonProps } from '../../compat/Button';
```